src/index.html

```diff
@@ -11,7 +11,213 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" onsubmit="onSubmit()">
+      <div class="block">
+        <fieldset>
+          <legend>Personal information:</legend>
+
+          <label for="surname" class="field-form">
+            Surname:
+
+            <input
+              type="text"
+              id="surname"
+              name="surname"
+              required
+              autocomplete="off"
+            >
+          </label>
+
+          <label for="name" class="field-form">
+            Name:
+
+            <input
+              type="text"
+              id="name"
+              name="name"
+              required
+              autocomplete="off"
+            >
+          </label>
+
+          <label for="age" class="field-form">
+            How old are You?
+
+            <input
+              type="number"
+              id="age"
+              name="age"
+              required
+              value="12"
+              min="0"
+              max="100"
+            >
+          </label>
+
+          <label for="date" class="field-form">
+            Full date of birth:
+
+            <input
+              type="date"
+              id="date"
+              name="date"
+            >
+          </label>
+
+          <label for="agr" class="field-form_without-margin">
+            I accept the term of the agreement
+
+            <input
+              type="checkbox"
+              id="agr"
+              name="agr"
```

Comment:
```suggestion
              name="terms"
```
Better to give more meaningful names. Like `terms` or `agreements`. It's hard to understand what `agr` should mean